DESCRIPTION

```diff
@@ -27,34 +27,30 @@ Authors@R:
       person(given = "2 Degrees Investing Initiative",
              role = c("cph", "fnd"),
              email = "contact@2degrees-investing.org"))
-Description: Provides a suite of metrics and analysis tools
-    commonly used for climate scenario analysis. For more information
-    visit <https://2degrees-investing.org/>.
-License: GPL-3 + file LICENSE
+Description: These tools help you to assess if a financial
+    portfolio aligns with climate goals. They summarize the production of
+    a financial portfolio and calculate targets for a given climate
+    scenario. These tools implement in R the last step of the free
+    software 'PACTA' (Paris Agreement Capital Transition Assessment;
+    <https://2degrees-investing.org/>). Financial institutions use 'PACTA'
+    to study how their capital allocation impacts the climate.
+License: GPL-3
```

Comment:
I would change the beginning to:
"These tools help you to assess if a financial portfolio aligns with climate goals. They summarize key metrics attributed to the portfolio (e.g. production, emission factors), and calculate targets based on climate scenarios. These tools implement in R the last step of the free software... etc."